packages/phase2cli/src/commands/setup.ts

```diff
@@ -76,21 +489,15 @@ import {
  */
 const setup = async () => {
     // Setup command state.
-    let circuitSequencePosition = 1 // The circuit's position for contribution.
-    let sharedCircomVersion: string = ""
-    let sharedCircomCommitHash: string = ""
-    let readyToSummarizeCeremony = false // Boolean flag to check whether the coordinator has finished to add circuits to the ceremony.
-
-    // Circuits.
-    const circuitsInputData: Array<CircuitInputData> = [] // All circuits input data.
-    const circuits: Array<CircuitDocument> = []
+    let circuitsInputData: Array<CircuitInputData> = [] // All circuits interactive data.
```

Comment:
Why are we defining this first and not directly as a constant on line 536?

Reply:
done